### ISSUE TITLE
API: Upload image

### ISSUE THREAD

**Issue by dignajar** [bludit/bludit#1009]
New feature for upload an image via API.

**Comment by dignajar** [comment]
@dimobelov
First approach. 
https://github.com/bludit/bludit/commit/2f63e920b07342ed787bf7172a88628e1a05abf0

**Comment by dignajar** [comment]
- API endpoint: http://localhost/api/images
- API `token`
- You need a user with administrator permissions for the `authentication` token

## Without uuid parameter
- Image directory `/bl-content/uploads/
- Thumbnail directory `/bl-content/uploads/thumbnail/

Request
```
curl -X POST \
  -F "token=eaf5df0a626145cc6d37b76f3eccc826" \
  -F "authentication=be2a8e536e401a43de0b49f8c456d277" \
  -F "image=@/home/diego/dog.jpg" \
  http://localhost/api/images
```

Response
```
{
  "status": "0",
  "message": "Image uploaded.",
  "image": "http://localhost/bl-content/uploads/dog.jpg",
  "thumbnail": "http://localhost/bl-content/uploads/thumbnails/dog.jpg"
}
```

## With uuid parameter
- Image directory `/bl-content/uploads/pages/<uuid>/`
- Thumbnail directory `/bl-content/uploads/pages/<uuid>/thumbnails/`

Request
```
curl -X POST \
  -F "token=eaf5df0a626145cc6d37b76f3eccc826" \
  -F "authentication=be2a8e536e401a43de0b49f8c456d277" \
  -F "uuid=c4ce04c04f8df45c97331d322eba83f7" \
  -F "image=@/home/diego/dog.jpg" \
  http://localhost/api/images
```

Response
```
{
  "status": "0",
  "message": "Image uploaded.",
  "filename": "http://localhost/bl-content/uploads/pages/c4ce04c04f8df45c97331d322eba83f7/dog.jpg",
  "thumbnail": "http://localhost/bl-content/uploads/pages/c4ce04c04f8df45c97331d322eba83f7/thumbnails/dog.jpg"
}
```

**Comment by dignajar** [comment]
Implemented in Bludit v3.9.0 beta 1
